src/components/calcite-panel/calcite-panel.tsx

```diff
@@ -85,9 +85,9 @@ export class CalcitePanel {
   @Prop({ reflect: true }) heightScale: CalciteScale;
 
   /**
-   * This sets width and max-width of the content area.
+   * This sets width and max-width of the panel.
```

Comment:
Nitpick: does this just state that it sets the width? I think `max-width` is more of an implementation detail.

Reply:
Copy that.